geos/entrypoints_test.go

```diff
@@ -767,3 +767,26 @@ func TestSymmetricDifference(t *testing.T) {
 
 	expectGeomEq(t, got, want, geom.IgnoreOrder)
 }
+
+func TestMakeValid(t *testing.T) {
+	for i, tt := range []struct {
+		input      string
+		wantOutput string
+	}{
+		{
+			"POLYGON((0 0,2 2,2 0,0 2,0 0))",
+			"MULTIPOLYGON(((0 0,0 2,1 1,0 0)),((2 0,1 1,2 2,2 0)))",
+		},
+	} {
+		t.Run(strconv.Itoa(i), func(t *testing.T) {
+			in := geomFromWKT(t, tt.input, geom.DisableAllValidations)
```

Comment:
Is there a way to assert that the `input` geometry is invalid?

Reply:
Yes, that's possible. Added in 65c283780294dd28c604144e5a1b32be7a024a28